### PR TITLE
Address grantr feedback on #149

### PR DESCRIPTION
**Pull request by evankanderson**
See #149

### CODE REVIEW

File: REPOSITORY-GUIDELINES.md [hunk 1]
```diff
@@ -114,33 +117,43 @@ Oversight Committee.
...
-bar:
+Working groups are allowed to request repositories that meet the following
+requirements. Actual repo creation will be performed by a SC or TOC member after
+verifying the requirements have been met:
 
-- Vendor independent (no direct links to specific clouds, no required
-  dependencies on vendor-specific tools)
+- Vendor dependencies must be clearly called out, and participation must be open
+  to the community.
+
```
Comment (mattmoor) [inline REPOSITORY-GUIDELINES.md:126]: Remove trailing whitespace:
```suggestion

```

File: REPOSITORY-GUIDELINES.md [hunk 2]
```diff
@@ -114,33 +117,43 @@ Oversight Committee.
...
+verifying the requirements have been met:
 
-- Vendor independent (no direct links to specific clouds, no required
-  dependencies on vendor-specific tools)
+- Vendor dependencies must be clearly called out, and participation must be open
+  to the community.
+  
+  - If the repository is for integration with a specific technology vendor,
+    affiliation with that vendor must not required for participation.
+
```
Comment (mattmoor) [inline REPOSITORY-GUIDELINES.md:129]: Remove trailing whitespace:
```suggestion

```

**Comment by grantr** [comment]
This PR should be rebased onto master. Only https://github.com/knative/community/pull/151/commits/4aa47dab41b3d2144ab984784d0c67c26ea95d10 is new; the other commits were merged in #149.

File: REPOSITORY-GUIDELINES.md [hunk 3]
```diff
@@ -93,19 +93,22 @@ repository is retired and no longer maintained.
...
 ## Additional / Experimental Repositories
 
-We want Knative working groups to be able to own code outside the core `knative`
-organization, which is kept intentionally small and has a high bar for entry.
-The `knative-sandbox` github organization was created for this purpose.
-Repositories in `knative-sandbox` are intended to give working groups more
-latitude to experiment with new ideas and to self-organize and manage
+It is beneficial for Knative working groups to be able to own code outside the
+core `knative` organization, which is kept intentionally small and has a high
+bar for entry.  The `knative-sandbox` github organization was created for this
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:98]: capitalization nit: `GitHub`

File: REPOSITORY-GUIDELINES.md [hunk 4]
```diff
@@ -93,19 +93,22 @@ repository is retired and no longer maintained.
...
 Note that `knative-sandbox` is **not** an incubation area for projects entering
-the core; in some cases, projects in the sandbox will remain there for the
-entire duration. Promotion from the sandbox into core will be handled on a
-case-by-case basis by joint decision of the Steering Committee and the Technical
-Oversight Committee.
+the core; in most cases, projects in the sandbox will remain there for the
+entire duration. `knative-sandbox` also provides a location for implementations
+of core interfaces which do not themselves need to live in the core, such as
+networking or eventing integrations. In the event that a working group wants a
+project in sandbox to be considered for transfor to the `knative` org, the
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:109]: typo nit: `transfer`

File: REPOSITORY-GUIDELINES.md [hunk 5]
```diff
@@ -114,33 +117,43 @@ Oversight Committee.
...
+requirements. Actual repo creation will be performed by a SC or TOC member after
+verifying the requirements have been met:
 
-- Vendor independent (no direct links to specific clouds, no required
-  dependencies on vendor-specific tools)
+- Vendor dependencies must be clearly called out, and participation must be open
+  to the community.
+  
+  - If the repository is for integration with a specific technology vendor,
+    affiliation with that vendor must not required for participation.
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:128]: typo nit: `must not be`

File: REPOSITORY-GUIDELINES.md [hunk 6]
```diff
@@ -114,33 +117,43 @@ Oversight Committee.
...
+    
+  - Non-integration repositories should aim to avoid dependencies on
+    vendor-specific technology by publishing abstract interface which can be
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:131]: typo nit: `abstract interfaces`

File: REPOSITORY-GUIDELINES.md [hunk 7]
```diff
@@ -114,33 +117,43 @@ Oversight Committee.
 Note: prior art here from the
 [Kubernetes community](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md).
 
-Working groups should be able to create repositories that meet the following
-bar:
+Working groups are allowed to request repositories that meet the following
+requirements. Actual repo creation will be performed by a SC or TOC member after
+verifying the requirements have been met:
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:122]: This section still suggests a chicken and egg problem but I defer to the suggestion in https://github.com/knative/community/pull/149#discussion_r441747631 that we split the mechanics out to another doc. #152 tracks this.

File: REPOSITORY-GUIDELINES.md [hunk 8]
```diff
@@ -172,15 +185,17 @@ The following are not required to create a working-group-owned repository:
...
+of the `knative-sandbox` organization and API groups to be renamed. This is
 intended to simplify the process in the common case, while giving Steering the
 ability to step in and rectify problems that may arise.
 
 ### Deletion
 
 Working groups are responsible for the upkeep of their repositories and are
-expected to keep a high quality bar. Inactive repositories should be archived,
-deleted, or moved out of the `knative-sandbox` organization.
+expected to keep a high quality bar. Inactive or non-confroming repositories
```
Comment (grantr) [inline REPOSITORY-GUIDELINES.md:195]: typo nit: `non-conforming`

File: REPOSITORY-GUIDELINES.md (hunk 3 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Done.

File: REPOSITORY-GUIDELINES.md (hunk 8 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Done.

File: REPOSITORY-GUIDELINES.md (hunk 6 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Done.

File: REPOSITORY-GUIDELINES.md (hunk 5 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Done.

File: REPOSITORY-GUIDELINES.md (hunk 7 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Tried rewording this again.

File: REPOSITORY-GUIDELINES.md (hunk 4 above)
Comment (evankanderson) [reply REPOSITORY-GUIDELINES.md]: Done.

**Comment by evankanderson** [comment]
/assign @pmorie 

I think this is basically clarifications, but it needs steering to approve since it's in the top-level. Since you've been driving these discussions, assigning to you.

**Comment by vaikas** [comment]
@pmorie just a ping if there's any changes still required here or if we're good to go.

**Comment by pmorie** [comment]
looking at this now, my sincere apologies for latency

**Review by pmorie — COMMENTED** [review]
Looks great, just a couple tiny little nits, wdyt?

**Comment by vaikas** [comment]
@duglin @ronavn @bsnchan 
To be discussed on 2021-03-04 meeting. Heads up so you get a chance to take a looksie.

**Comment by bsnchan** [comment]
TM committee spoke about this issue at our TM meeting on [March 4th](https://docs.google.com/document/d/1ghGQl0O7TM8N_YQ33w6xKyQ2yUnOX0ACsOnlqoV_XgY/edit?ts=604132d0) - we decided that the TM committee only really care about the "specs" repo and that structure of repos/ref implementation is out of scope for TM committee.

cc @knative/trademark-committee

**Review by evankanderson — COMMENTED** [review]
(Forgot to send these comments, but wanted to enter them for the record.)

**Pull request merged**